tokens/blocket.se/avatar.yml

```diff
@@ -3,5 +3,5 @@ token: maps
 
 color:
   avatar:
-    background: s-color-background-subtle
-    icon: s-color-icon-subtle
+    background: bluegray-100
```

Comment:
Are the blocket tokens supposed to be the same as FINN?

Reply:
Yes, for now... there are no colors set for Blocket yet. 🙂

Reply:
...so it really doesn't matter. 😄